nodebuilder/p2p/pubsub.go

```diff
@@ -109,12 +115,21 @@ func topicScoreParams(network Network) map[string]*pubsub.TopicScoreParams {
 	return mp
 }
 
-func peerScoreParams(isBootstrapper bool, bootstrappers Bootstrappers) *pubsub.PeerScoreParams {
+func peerScoreParams(isBootstrapper bool, bootstrappers Bootstrappers, cfg Config) (*pubsub.PeerScoreParams, error) {
 	bootstrapperSet := map[peer.ID]struct{}{}
 	for _, b := range bootstrappers {
 		bootstrapperSet[b.ID] = struct{}{}
 	}
 
+	ipColocFactWl := make([]*net.IPNet, 0)
```

Comment:
It's not a finished preallocation
```suggestion
	ipColocFactWl := make([]*net.IPNet, 0, len(cfg.IPColocationWhitelist))
```